### PR TITLE
Support docs revised

### PR DESCRIPTION
**Pull request by sstrubberg**
Based on a conversation earlier today with @laurenmrice and @tay1orjones, we have updated our support docs to better clarify what "being on support" looks like.

### What's changed

- New section on Slack support
- New section for Slack/Github integration support
- Added the kit repo to the repos supported
- Added nuance to the types of issues
- Added a section on our GH Project and what the categories mean.

### Reviewing

Please give feedback where your experience differs on how you deliver support, bringing together all the best practices.

### CODE REVIEW

File: docs/guides/support.md [hunk 1]
```diff
@@ -29,22 +30,73 @@
...
 ## Overview
 
-Ongoing support for Carbon and previous version (which will be prioritized in
-that order)
+Ongoing support for the latest version Carbon and the previous version (which
+will be prioritized in that order)
+
+Whether you are new or a seasoned veteran, being a maintainer of the Carbon
+Design System can be overwhelming and can often lead to feelings of imposter
+syndrome. Whether you are triaging new issues in Github or responding to Slack
```
Comment (laurenmrice) [inline docs/guides/support.md]:
```suggestion
syndrome. Whether you are triaging new issues in GitHub or responding to Slack
```

File: docs/guides/support.md [hunk 2]
```diff
@@ -29,22 +30,73 @@
...
+threads in our support channels, here are some responses that can help get the
+conversation started in case you get stuck.
+
+- “Could you share more about you’ve already tried?”
```
Comment (laurenmrice) [inline docs/guides/support.md]:
```suggestion
- “Could you share more about what you’ve already tried?”
```

File: docs/guides/support.md [hunk 3]
```diff
@@ -29,22 +30,73 @@
...
+- “Can you share more about your use-case?”
```
Comment (laurenmrice) [inline docs/guides/support.md]:
```suggestion
- “Can you share more about your usecase?”
```

File: docs/guides/support.md [hunk 4]
```diff
@@ -29,22 +30,73 @@
...
 - Does not include feature enhancements or new feature requests
 
+### What does "ongoing support" mean regarding Slack?
+
+- A minimum of twice per day that maintainers monitor #carbon-design-system and
+  #carbon-react Slack support channels and respond to questions. Help our users
+  find solutions to their problems.
+- The goal is to _respond_ to every thread on your day.
+- Go into threads where members of the community answer a question and validate
+  the person in the community that answered it by adding an emojii or response
```
Comment (laurenmrice) [inline docs/guides/support.md]:
```suggestion
  the person in the community that answered it by adding an emoji or response
```

File: docs/guides/support.md [hunk 5]
```diff
@@ -29,22 +30,73 @@
...
+  reiterating the answer and acknowledging the community member.
+- Use discretion to respond to applicable threads. Lean on the maintainer team's
+  private #carbon-support channel to look for subject matter experts when a
+  question goes beyond your skillset. When you ask in this channel, take a leap
+  of faith in your skills and propose a response, such as, "This is how I want
+  to respond ... YOUR REPLY HERE". Follow the question by practicing humility,
+  "How far off am I"? What am I missing? Remember, you're acting as an advocate
```
Comment (laurenmrice) [inline docs/guides/support.md]:
```suggestion
  "How far off am I"? "What am I missing"? Remember, you're acting as an advocate
```

File: docs/guides/support.md [hunk 6]
```diff
@@ -29,22 +30,73 @@
...
+Whether you are new or a seasoned veteran, being a maintainer of the Carbon
+Design System can be overwhelming and can often lead to feelings of imposter
+syndrome. Whether you are triaging new issues in Github or responding to Slack
+threads in our support channels, here are some responses that can help get the
+conversation started in case you get stuck.
+
+- “Could you share more about you’ve already tried?”
+- “Can you share more about your use-case?”
+- “Can you add a repro (reproduction) using Stackblitz?”
+- “After searching through FILL IN RESOURCE HERE, I was able to find this and
```
Comment (laurenmrice) [inline docs/guides/support.md]: Just a suggestion: When saying ...FILL IN RESOURCE HERE, you could also put it in brackets like this [FILL IN RESOURCE HERE]

File: docs/guides/support.md [hunk 7]
```diff
@@ -29,22 +30,73 @@
...
+- Go into threads where members of the community answer a question and validate
+  the person in the community that answered it by adding an emojii or response
+  reiterating the answer and acknowledging the community member.
+- Use discretion to respond to applicable threads. Lean on the maintainer team's
+  private #carbon-support channel to look for subject matter experts when a
+  question goes beyond your skillset. When you ask in this channel, take a leap
+  of faith in your skills and propose a response, such as, "This is how I want
+  to respond ... YOUR REPLY HERE". Follow the question by practicing humility,
+  "How far off am I"? What am I missing? Remember, you're acting as an advocate
+  for the person who originally asked the question. Respresent their needs to
```
Comment (laurenmrice) [inline docs/guides/support.md]:
```suggestion
  for the person who originally asked the question. Represent their needs to
```

File: docs/guides/support.md [hunk 8]
```diff
@@ -29,22 +30,73 @@
...
+  the best of your ability.
+- If a question is Carbon-related, but not necessarily related to the core
+  components, route the person to the correct Slack channel where they might
+  receive support.
+  - [ibmproducts-pal-dev](https://ibm.enterprise.slack.com/archives/C013ZTX0N6B)
+  - [ibmproducts-pal](https://ibm.enterprise.slack.com/archives/CQGR0HC05)
+  - [carbon-angulur](https://ibm.enterprise.slack.com/archives/CBZC0LM2N)
```
Comment (laurenmrice) [inline docs/guides/support.md]:
```suggestion
  - [carbon-ng](https://ibm.enterprise.slack.com/archives/CBZC0LM2N)
```

File: docs/guides/support.md [hunk 9]
```diff
@@ -29,22 +30,73 @@
...
+  - [carbon-for-cloud](https://ibm.enterprise.slack.com/archives/C0603NZDXRD)
+  - [carbon-charts](https://ibm.enterprise.slack.com/archives/CCA7L4MS9)
+  - [carbon-for-ibm-dotcom](https://ibm.enterprise.slack.com/archives/C2PLX8GQ6)
+  - [carbon-web-components](https://ibm.enterprise.slack.com/archives/CL83LMKSA)
+  - [carbon-wg-typescript](https://ibm.enterprise.slack.com/archives/C03C8VASVED)
+
+### Configuring the Github Slack integration
```
Comment (laurenmrice) [inline docs/guides/support.md]:
```suggestion
### Configuring the GitHub Slack integration
```

File: docs/guides/support.md [hunk 10]
```diff
@@ -64,14 +116,20 @@ to define the issue their having._
 
 Using these templates also helps Project-Bot automatically move issues and new
 pull requests to the right workstreams by auto labeling the issues according to
-the template type. There are three types of issues The Support and Maintenance
-team are concerned with:
+the template type. There are three types of issues the maintainer team are
```
Comment (laurenmrice) [inline docs/guides/support.md]:
```suggestion
the template type. There are three types of issues the maintainer team is
```

File: docs/guides/support.md [hunk 11]
```diff
@@ -374,6 +428,28 @@ workaround, a
 https://github.com/carbon-design-system/carbon/labels/severity%3A%202 or lower
 is more appropriate.
 
+After applying the severity label, be sure to also apply the appropriate role to
+the issue. Either `role: dev 🤖` or `role: design :pencil2:`, depending on which
```
Comment (laurenmrice) [inline docs/guides/support.md:430]: I would double check the pencil emoji here to make sure its showing up.

File: docs/guides/support.md [hunk 12]
```diff
@@ -374,6 +428,28 @@ workaround, a
...
+discipline should be working on it.
+
+### Project
+
+After assigning the role and serverity, be sure to apply the correct categories
```
Comment (laurenmrice) [inline docs/guides/support.md]:
```suggestion
After assigning the role and severity, be sure to apply the correct categories
```

File: docs/guides/support.md [hunk 13]
```diff
@@ -374,6 +428,28 @@ workaround, a
...
+in the Github Project:
```
Comment (laurenmrice) [inline docs/guides/support.md]:
```suggestion
in the GitHub Project:
```

File: docs/guides/support.md [hunk 14]
```diff
@@ -374,6 +428,28 @@ workaround, a
...
+
+- Status
+  - Triage - Issues that need triaging
+  - Needs Refined - Issues that will be worked on in future sprints, but need to
+    be broken down into smaller chunks and discussed.
+  - Needs a11y review - Issues that need to be brough the Carbon Accessibility
```
Comment (laurenmrice) [inline docs/guides/support.md]:
```suggestion
  - Needs a11y review - Issues that need to be brought to the Carbon Accessibility
```

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Looks good! Just found a couple typos

**Review by andreancardona — APPROVED** [review]
looks good! would recommend @aubrey-oneal to review

**Review by guidari — APPROVED** [review]
Awesome! The docs looks good to me 🚀

One thing that also might be good is implementing bookmarks in the 'carbon-support' channel. For example the Stackblitz draft for reproduction examples and etc...

Screenshot 2023-11-16 at 09 08 30

I can add a few that I find interesting to be there.

**Review by laurenmrice — APPROVED** [review]
Looks good Scott!

**Comment by sstrubberg** [comment]
> Awesome! The docs looks good to me 🚀
> 
> One thing that also might be good is implementing bookmarks in the 'carbon-support' channel. For example the Stackblitz draft for reproduction examples and etc...
> 
> Screenshot 2023-11-16 at 09 08 30
> 
> I can add a few that I find interesting to be there.

that's a fantastic idea. Please add them!

**Review by aubrey-oneal — APPROVED** [review]
Looks great! 🚀

**Pull request merged**